packages/angular/projects/dev/src/app/datagrid/pagination/pagination.ts

```diff
@@ -25,8 +25,8 @@ export class DatagridPaginationDemo {
     this.users = inventory.all;
   }
 
-  pageChange($event: Event) {
-    console.log($event);
+  pageChange(pageNumber: number) {
+    console.log(pageNumber);
```

Comment:
Do we need this `console.log`?

Reply:
This is for demo purpose inside the devApp, so yes :)